### PR TITLE
Show list city

### PR DESCRIPTION
**Pull request by at-tringuyen3**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Task ID: TI-18

## Reference Isuees
#18

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: app/Http/Controllers/CityController.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    /**
+     ** Create contructor.
+     *
+     * @param App\Models\City $city city
+     *
+     * @return void
+     */
+    public function __construct(City $city)
+    {
+        $this->City = $city;
```
Comment (at-tuannguyen3) [inline app/Http/Controllers/CityController.php]: $this->city chứ ko phỉa $this->City

File: app/Http/Controllers/CityController.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $cities = $this->City->getCities();
```
Comment (at-tuannguyen3) [inline app/Http/Controllers/CityController.php]: đây sửa luôn lại nhé

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment!

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**